package/common_init/overlay/etc/init.d/S86health_daemon

```diff
@@ -5,4 +5,10 @@ cmd="health_daemon"
 dir="/"
 user="healthd"
 
+setup_permissions() {
+  mkdir -p /var/run/health
+  echo '0' >> /var/run/health/gps_time_available
```

Comment:
Using > here is more appropriate

Reply:
Thanks, I am not seasoned bash script writer so be aware for these simple things.